server/swap/swap.go

```diff
@@ -2028,8 +2024,8 @@ func readMatches(matchSets []*order.MatchSet, feeRates map[uint32]uint64) []*mat
 				takerSwapAsset = base
 			}
 
-			match.FeeRateBase = feeRates[base]
-			match.FeeRateQuote = feeRates[quote]
+			match.FeeRateBase = matchSet.FeeRateBase
+			match.FeeRateQuote = matchSet.FeeRateQuote
```

Comment:
I think `matchSet.Matches()` -> `dex/order.newMatch` has this job now, so redundant I believe.